frontend/src/modules/member/components/view/member-view-aside.vue

```diff
@@ -28,17 +28,11 @@
           "
           target="_blank"
         >
-          <div>
-            <span
-              class="btn cursor-auto p-2 bg-gray-100 border border-gray-200 mr-3"
-              :class="`btn--${platform}`"
-            >
-              <img
-                :src="findIcon(platform)"
-                :alt="`${platform}-icon`"
-                class="w-4 h-4"
-              />
-            </span>
+          <div class="flex gap-3 items-center">
+            <app-platform
+              :platform="platform"
+              :platform-name="`${platform}-icon`"
```

Comment:
Shouldn't `platform-name` be a readable name like `GitHub` or `StackOverflow`?

Reply:
Since I was using `platform-name` for both the tracking event names and the alt property of the images it didn't have to be readable. But to prevent errors and confusion I removed this prop and added one for the track events:
`trackEventName`, if passed we will track the clicks

For the alt property, I'm now just getting the name value of the integrations json from the platform value